src/components/FetchHandler.tsx

```diff
@@ -27,28 +25,27 @@ export const FetchHandler = <TResult extends {}>({
     notFoundIndicator,
     loadingIndicator,
 }: Props<TResult>) => {
-    const { loading, data, error, trigger, errorCode } = usePromise(apiMethod);
```

Comment:
Varför tar du bort errorCode från denna listan? Det skadar väl inte att ha kvar om man vill ta tag i den specifikt?

Reply:
Well, @unger created usePromiseIncludingErrorData in some other project and we decided to implement this in the framework. Also, you still get the error code anyways.

Reply:
"Why are you removing errorCode from this list? It doesn't hurt to have one left if you want to grab it specifically?"